models/fieldtype_test.go

```diff
@@ -0,0 +1,95 @@
+package models
+
+import "testing"
+
+var stString = SimpleType{KindString}
+var stInt = SimpleType{KindInteger}
```

Comment:
Variable declaration can be factored into blocks. e.g., https://tour.golang.org/basics/11